### PR TITLE
Updates docs to include information about the new actions

### PR DESCRIPTION
**Pull request by claudiubelu**
We now have 2 models deployed on the EKS environment, one for staging, and one for production. The actions have been updated to address this, and the docs needed to be updated. Additional actions have also been added.

Added documentation for the new actions.

### CODE REVIEW

File: docs/repository_jobs.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# FINOS Legend Juju EKS deployment - Periodic refresh
+
+The [Refresh EKS FINOS Legend deployment](../.github/workflows/scheduled.yaml) job will run periodically for a configured Juju EKS deployment and will refresh the Legend Engine, Studio, and SDLC charms on the Staging environment to their latest revisions (including the Legend docker images associated the revisions). This job requires a few Secrets to be defined in the project repository (``Settings > Secrets > Actions > Repository secrets``):
```
Comment (fabi200123) [inline docs/repository_jobs.md:3]: associated `to` the revision

File: docs/repository_jobs.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+- ``KUBECONFIG_B64``: The EKS kubeconfig file in base64 format. The kubeconfig file is typically found in ``~/.kube/config``, and to can be transformed into base64 by running: ``cat ~/.kube/config | base64``.
```
Comment (fabi200123) [inline docs/repository_jobs.md:5]: and `it` can be

File: docs/repository_jobs.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- ``CONTROLLERS_B64``: The Juju ``controllers.yaml`` file in base64 format. The file is typically found in ``~/.local/share/juju/controllers.yaml``. The file will contain information about all the locally known Juju Controllers, but only the one containing the Legend model is needed. To transform it into base64, run: ``cat ~/.local/share/juju/controllers.yaml | base64``.
+- ``CONTROLLER_PASSWORD``: The Juju Controller's admin password. It can be typically found found in ``~/.local/share/juju/accounts.yaml``. This password will be used to authenticate into the Juju Controller.
```
Comment (fabi200123) [inline docs/repository_jobs.md:7]: Only one `found`

File: docs/repository_jobs.md (hunk 1 above)
Comment (claudiubelu) [reply docs/repository_jobs.md:3]: Done

File: docs/repository_jobs.md (hunk 2 above)
Comment (claudiubelu) [reply docs/repository_jobs.md:5]: Done

File: docs/repository_jobs.md (hunk 3 above)
Comment (claudiubelu) [reply docs/repository_jobs.md:7]: Done

**Review by maoo — APPROVED** [review]
Lgtm

**Pull request merged**